docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+....
+const config = withModuleFederationPlugin({
+  name: 'example-ui',
+  filename: 'remoteEntry.js',
+  exposes: {
+    './ExampleComponent': './src/app/remotes/example/example.component.main.ts'
+  },
+  ...
+})
+....
+
+Webpack configuration is exposing an **example.component.main.ts** file.
+
+.src/app/remotes/example/example.component.main.ts
+[source, typescript]
+....
+import('./example.component.bootstrap').catch((err) => console.error(err))
+....
+
+The **example.component.main.ts** file is importing (asynchronously) the **example.component.bootstrap.ts** file.
+
+.src/app/remotes/example/example.component.bootstrap.ts
+[source, typescript]
+....
+import {
+  HttpClient,
+  provideHttpClient,
+  withInterceptorsFromDi,
+} from '@angular/common/http';
+import {
+  APP_INITIALIZER,
+  importProvidersFrom
+} from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { AngularAuthModule } from '@onecx/angular-auth';
+import { bootstrapRemoteComponent } from '@onecx/angular-webcomponents';
+import {
+  createRemoteComponentTranslateLoader,
+  UserService
+} from '@onecx/portal-integration-angular';
+import { environment } from 'src/environments/environment';
+import { ExampleComponent } from './example.component';
+import {
+  BASE_URL,
+  provideTranslateServiceForRoot,
+} from '@onecx/angular-remote-components';
+import { TranslateLoader } from '@ngx-translate/core';
+import { ReplaySubject } from 'rxjs';
+
+function userProfileInitializer(userService: UserService) {
+  return async () => {
+    await userService.isInitialized;
+  };
+}
+
+bootstrapRemoteComponent(
+  ExampleComponent,
+  'example-remote-component',
+  environment.production,
+  [
+    provideHttpClient(withInterceptorsFromDi()),
+    {
+      provide: BASE_URL,
+      useValue: new ReplaySubject<string>(1),
+    },
+    provideTranslateServiceForRoot({
+      isolate: true,
+      loader: {
+        provide: TranslateLoader,
+        useFactory: createRemoteComponentTranslateLoader,
+        deps: [HttpClient, BASE_URL],
+      },
+    }),
+    importProvidersFrom(
+      AngularAuthModule,
+      BrowserModule,
+      BrowserAnimationsModule,
+    ),
+    {
+      provide: APP_INITIALIZER,
+      useFactory: userProfileInitializer,
+      deps: [UserService],
+      multi: true,
+    },
+  ]
+)
+....
+
+// TODO: Correct link to exact resource for Angular issues
+// TODO: Link to the @onecx/angular-webcomponents
+The **example.component.bootstrap.ts** file is bootstrapping the Remote Component using @onecx/angular-webcomponents library to ensure that Angular ngZone and platform are shared (Angular requirement only). It also connects Shell router with the Remote Component's router (if such exists). The last argument is an array of providers required for the component to work properly. Detailed description of this file can be found in the <<summary>>.
```

Comment:
It also connects **the** Shell router with the Remote Component's router (if such exists)...